diff --git a/src/db/schema.rs b/src/db/schema.rs
index d105e29..7773c57 100644
--- a/src/db/schema.rs
+++ b/src/db/schema.rs
@@ -11,7 +11,7 @@ diesel::table! {
 
 diesel::table! {
     backfill_jobs (chain_id, from_block, to_block) {
-        address -> Array<Nullable<Bytea>>,
+        address -> Array<Bytea>,
         chain_id -> Int4,
         from_block -> Int4,
         to_block -> Int4,
@@ -42,9 +42,4 @@ diesel::table! {
